{
 "errors": {
  "pkg_resources": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/__init__.py'",
    ""
   ]
  },
  "pkg_resources._vendor": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/__init__.py'",
    ""
   ]
  },
  "pkg_resources._vendor.appdirs": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/appdirs.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/appdirs.py'",
    ""
   ]
  },
  "pkg_resources._vendor.packaging": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/__init__.py'",
    ""
   ]
  },
  "pkg_resources._vendor.packaging.__about__": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/__about__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/__about__.py'",
    ""
   ]
  },
  "pkg_resources._vendor.packaging._compat": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/_compat.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/_compat.py'",
    ""
   ]
  },
  "pkg_resources._vendor.packaging._structures": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/_structures.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/_structures.py'",
    ""
   ]
  },
  "pkg_resources._vendor.packaging.markers": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/markers.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/markers.py'",
    ""
   ]
  },
  "pkg_resources._vendor.packaging.requirements": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/requirements.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/requirements.py'",
    ""
   ]
  },
  "pkg_resources._vendor.packaging.specifiers": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/specifiers.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/specifiers.py'",
    ""
   ]
  },
  "pkg_resources._vendor.packaging.utils": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/utils.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/utils.py'",
    ""
   ]
  },
  "pkg_resources._vendor.packaging.version": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/version.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/packaging/version.py'",
    ""
   ]
  },
  "pkg_resources._vendor.pyparsing": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/pyparsing.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/pyparsing.py'",
    ""
   ]
  },
  "pkg_resources._vendor.six": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/six.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/_vendor/six.py'",
    ""
   ]
  },
  "pkg_resources.extern": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/extern/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/extern/__init__.py'",
    ""
   ]
  },
  "pkg_resources.py31compat": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/py31compat.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/pkg_resources/py31compat.py'",
    ""
   ]
  },
  "setuptools.command": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/__init__.py'",
    ""
   ]
  },
  "setuptools.command.alias": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/alias.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/alias.py'",
    ""
   ]
  },
  "setuptools.command.bdist_rpm": {
   "exception": "write to closed file",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/helpers.py\", line 488, in wrapper",
    "    return func(self, line, column, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 213, in complete",
    "    return completion.complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 169, in complete",
    "    cached_name, completion_names = self._complete_python(leaf)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 310, in _complete_python",
    "    completion_names += self._complete_global_scope()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 376, in _complete_global_scope",
    "    completion_names += filter.values()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/filters.py\", line 93, in values",
    "    for name_key in self._used_names",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/filters.py\", line 88, in _convert_names",
    "    return [self.name_class(self.parent_context, name) for name in names]",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/filters.py\", line 88, in <listcomp>",
    "    return [self.name_class(self.parent_context, name) for name in names]",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/filters.py\", line 95, in <genexpr>",
    "    _get_definition_names(self._used_names, name_key),",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/filters.py\", line 122, in _filter",
    "    return list(self._check_flows(names))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/filters.py\", line 137, in _check_flows",
    "    origin_scope=self._origin_scope",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/flow_analysis.py\", line 85, in reachability_check",
    "    return _break_check(context, value_scope, first_flow_scope, node)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/flow_analysis.py\", line 93, in _break_check",
    "    reachable = _check_if(context, check_node)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/flow_analysis.py\", line 120, in _check_if",
    "    types = context.infer_node(node)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 222, in infer_node",
    "    return infer_node(self, node)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 156, in infer_node",
    "    return _infer_node_if_inferred(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 169, in _infer_node_if_inferred",
    "    return _infer_node_cached(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 174, in _infer_node_cached",
    "    return _infer_node(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/debug.py\", line 81, in wrapper",
    "    return func(*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 82, in wrapper",
    "    return func(context, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 254, in _infer_node",
    "    return infer_or_test(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 460, in infer_or_test",
    "    types = context.infer_node(next(iterator))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 222, in infer_node",
    "    return infer_node(self, node)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 156, in infer_node",
    "    return _infer_node_if_inferred(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 169, in _infer_node_if_inferred",
    "    return _infer_node_cached(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 174, in _infer_node_cached",
    "    return _infer_node(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/debug.py\", line 81, in wrapper",
    "    return func(*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 82, in wrapper",
    "    return func(context, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 197, in _infer_node",
    "    value_set = context.infer_node(first_child)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 222, in infer_node",
    "    return infer_node(self, node)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 156, in infer_node",
    "    return _infer_node_if_inferred(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 169, in _infer_node_if_inferred",
    "    return _infer_node_cached(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 174, in _infer_node_cached",
    "    return _infer_node(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/debug.py\", line 81, in wrapper",
    "    return func(*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 82, in wrapper",
    "    return func(context, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 184, in _infer_node",
    "    return infer_atom(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 304, in infer_atom",
    "    return context.py__getattribute__(atom, position=position)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 75, in py__getattribute__",
    "    values = ValueSet.from_sets(name.infer() for name in names)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/base_value.py\", line 426, in from_sets",
    "    for set_ in sets:",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 75, in <genexpr>",
    "    values = ValueSet.from_sets(name.infer() for name in names)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/names.py\", line 284, in infer",
    "    self.tree_name",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/__init__.py\", line 21, in wrapper",
    "    return built_functions[public_name](*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/stdlib.py\", line 867, in wrapper",
    "    return func(inference_state, context, tree_name)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/django.py\", line 177, in wrapper",
    "    result = func(inference_state, context, tree_name)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 744, in tree_name_to_values",
    "    types = imports.infer_import(context, tree_name)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 53, in infer_import",
    "    _prepare_infer_import(module_context, tree_name)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 116, in _prepare_infer_import",
    "    return from_import_name, tuple(import_path), import_node.level, importer.follow()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 297, in follow",
    "    self._inference_state, self.import_path, sys_path, self._module_context",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 383, in import_module_by_names",
    "    ) for parent_module_value in value_set",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 383, in <listcomp>",
    "    ) for parent_module_value in value_set",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/__init__.py\", line 21, in wrapper",
    "    return built_functions[public_name](*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/flask.py\", line 20, in wrapper",
    "    return callback(inference_state, import_names, module_context, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 119, in wrapper",
    "    for p in parent_module_values",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/base_value.py\", line 426, in from_sets",
    "    for set_ in sets:",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 119, in <genexpr>",
    "    for p in parent_module_values",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 415, in import_module",
    "    is_global_search=True,",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 133, in wrapper",
    "    kwargs=kwargs,",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 226, in run",
    "    return self._send(id(inference_state), function, args, kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 241, in _send",
    "    pickle_dump(data, self._get_process().stdin, PICKLE_PROTOCOL)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/_compatibility.py\", line 39, in pickle_dump",
    "    pickle.dump(data, file, protocol)",
    "ValueError: write to closed file",
    ""
   ]
  },
  "setuptools.command.bdist_wininst": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/bdist_wininst.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/bdist_wininst.py'",
    ""
   ]
  },
  "setuptools.command.build_ext": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/build_ext.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/build_ext.py'",
    ""
   ]
  },
  "setuptools.command.build_py": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/build_py.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/build_py.py'",
    ""
   ]
  },
  "setuptools.command.develop": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/develop.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/develop.py'",
    ""
   ]
  },
  "setuptools.command.easy_install": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/easy_install.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/easy_install.py'",
    ""
   ]
  },
  "setuptools.command.egg_info": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/egg_info.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/egg_info.py'",
    ""
   ]
  },
  "setuptools.command.install": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/install.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/install.py'",
    ""
   ]
  },
  "setuptools.command.install_egg_info": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/install_egg_info.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/install_egg_info.py'",
    ""
   ]
  },
  "setuptools.command.install_lib": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/install_lib.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/install_lib.py'",
    ""
   ]
  },
  "setuptools.command.install_scripts": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/install_scripts.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/install_scripts.py'",
    ""
   ]
  },
  "setuptools.command.py36compat": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/py36compat.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/py36compat.py'",
    ""
   ]
  },
  "setuptools.command.register": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/register.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/register.py'",
    ""
   ]
  },
  "setuptools.command.rotate": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/rotate.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/rotate.py'",
    ""
   ]
  },
  "setuptools.command.saveopts": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/saveopts.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/saveopts.py'",
    ""
   ]
  },
  "setuptools.command.sdist": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/sdist.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/sdist.py'",
    ""
   ]
  },
  "setuptools.command.setopt": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/setopt.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/setopt.py'",
    ""
   ]
  },
  "setuptools.command.test": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/test.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/test.py'",
    ""
   ]
  },
  "setuptools.command.upload": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/upload.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/upload.py'",
    ""
   ]
  },
  "setuptools.command.upload_docs": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/upload_docs.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/command/upload_docs.py'",
    ""
   ]
  },
  "setuptools.extern": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/extern/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpqre7naoz/lib/python2.7/site-packages/setuptools/extern/__init__.py'",
    ""
   ]
  }
 },
 "symbols": [
  "easy_install.main",
  "setuptools.Command",
  "setuptools.Distribution",
  "setuptools.Extension",
  "setuptools.Feature",
  "setuptools.PEP420PackageFinder",
  "setuptools.PackageFinder",
  "setuptools.Require",
  "setuptools._Command",
  "setuptools.__all__",
  "setuptools.__version__",
  "setuptools._find_all_simple",
  "setuptools.archive_util",
  "setuptools.archive_util.UnrecognizedFormat",
  "setuptools.archive_util.__all__",
  "setuptools.archive_util.default_filter",
  "setuptools.archive_util.extraction_drivers",
  "setuptools.archive_util.unpack_archive",
  "setuptools.archive_util.unpack_directory",
  "setuptools.archive_util.unpack_tarfile",
  "setuptools.archive_util.unpack_zipfile",
  "setuptools.bootstrap_install_from",
  "setuptools.command",
  "setuptools.command.bdist_egg.Command",
  "setuptools.command.bdist_egg.INSTALL_DIRECTORY_ATTRS",
  "setuptools.command.bdist_egg.Library",
  "setuptools.command.bdist_egg.NATIVE_EXTENSIONS",
  "setuptools.command.bdist_egg._get_purelib",
  "setuptools.command.bdist_egg.analyze_egg",
  "setuptools.command.bdist_egg.bdist_egg",
  "setuptools.command.bdist_egg.can_scan",
  "setuptools.command.bdist_egg.iter_symbols",
  "setuptools.command.bdist_egg.make_zipfile",
  "setuptools.command.bdist_egg.safety_flags",
  "setuptools.command.bdist_egg.scan_module",
  "setuptools.command.bdist_egg.six",
  "setuptools.command.bdist_egg.strip_module",
  "setuptools.command.bdist_egg.walk_egg",
  "setuptools.command.bdist_egg.write_safety_flag",
  "setuptools.command.bdist_egg.write_stub",
  "setuptools.command.build_clib.build_clib",
  "setuptools.command.build_clib.newer_pairwise_group",
  "setuptools.command.easy_install.main",
  "setuptools.config",
  "setuptools.config.ConfigHandler",
  "setuptools.config.ConfigMetadataHandler",
  "setuptools.config.ConfigOptionsHandler",
  "setuptools.config.configuration_to_dict",
  "setuptools.config.import_module",
  "setuptools.config.parse_configuration",
  "setuptools.config.read_configuration",
  "setuptools.config.string_types",
  "setuptools.dep_util",
  "setuptools.dep_util.newer_pairwise_group",
  "setuptools.depends",
  "setuptools.depends.Require",
  "setuptools.depends.__all__",
  "setuptools.depends._update_globals",
  "setuptools.depends.extract_constant",
  "setuptools.depends.find_module",
  "setuptools.depends.get_module_constant",
  "setuptools.dist",
  "setuptools.dist.Distribution",
  "setuptools.dist.Feature",
  "setuptools.dist.Require",
  "setuptools.dist._Distribution",
  "setuptools.dist.__all__",
  "setuptools.dist._check_extra",
  "setuptools.dist._get_unpatched",
  "setuptools.dist.assert_bool",
  "setuptools.dist.assert_string_list",
  "setuptools.dist.check_entry_points",
  "setuptools.dist.check_extras",
  "setuptools.dist.check_importable",
  "setuptools.dist.check_nsp",
  "setuptools.dist.check_package_data",
  "setuptools.dist.check_packages",
  "setuptools.dist.check_requirements",
  "setuptools.dist.check_specifier",
  "setuptools.dist.check_test_suite",
  "setuptools.dist.filter",
  "setuptools.dist.filterfalse",
  "setuptools.dist.get_unpatched",
  "setuptools.dist.map",
  "setuptools.dist.parse_configuration",
  "setuptools.dist.sequence",
  "setuptools.dist.six",
  "setuptools.dist.windows_support",
  "setuptools.dist.write_pkg_file",
  "setuptools.dist.write_pkg_info",
  "setuptools.extension",
  "setuptools.extension.Extension",
  "setuptools.extension.Library",
  "setuptools.extension._Extension",
  "setuptools.extension._have_cython",
  "setuptools.extension.have_pyrex",
  "setuptools.extension.map",
  "setuptools.extern",
  "setuptools.extern.six",
  "setuptools.extern.six.binary_type",
  "setuptools.extern.six.moves.builtins",
  "setuptools.extern.six.moves.configparser",
  "setuptools.extern.six.moves.filter",
  "setuptools.extern.six.moves.filterfalse",
  "setuptools.extern.six.moves.http_client",
  "setuptools.extern.six.moves.map",
  "setuptools.extern.six.moves.urllib",
  "setuptools.extern.six.string_types",
  "setuptools.filter",
  "setuptools.find_packages",
  "setuptools.findall",
  "setuptools.glob",
  "setuptools.glob.__all__",
  "setuptools.glob._iglob",
  "setuptools.glob._isrecursive",
  "setuptools.glob._rlistdir",
  "setuptools.glob.binary_type",
  "setuptools.glob.escape",
  "setuptools.glob.glob",
  "setuptools.glob.has_magic",
  "setuptools.glob.iglob",
  "setuptools.glob.magic_check",
  "setuptools.glob.magic_check_bytes",
  "setuptools.launch",
  "setuptools.launch.run",
  "setuptools.lib2to3_ex",
  "setuptools.lib2to3_ex.DistutilsRefactoringTool",
  "setuptools.lib2to3_ex.Mixin2to3",
  "setuptools.lib2to3_fixer_packages",
  "setuptools.map",
  "setuptools.monkey",
  "setuptools.monkey.__all__",
  "setuptools.monkey._get_mro",
  "setuptools.monkey._patch_distribution_metadata_write_pkg_file",
  "setuptools.monkey._patch_distribution_metadata_write_pkg_info",
  "setuptools.monkey.get_unpatched",
  "setuptools.monkey.patch_all",
  "setuptools.monkey.patch_for_msvc_specialized_compiler",
  "setuptools.monkey.patch_func",
  "setuptools.monkey.six",
  "setuptools.msvc",
  "setuptools.msvc.EnvironmentInfo",
  "setuptools.msvc.PlatformInfo",
  "setuptools.msvc.RegistryInfo",
  "setuptools.msvc.SystemInfo",
  "setuptools.msvc._augment_exception",
  "setuptools.msvc._msvc9_suppress_errors",
  "setuptools.msvc.filterfalse",
  "setuptools.msvc.msvc14_gen_lib_options",
  "setuptools.msvc.msvc14_get_vc_env",
  "setuptools.msvc.msvc9_find_vcvarsall",
  "setuptools.msvc.msvc9_query_vcvarsall",
  "setuptools.msvc.safe_env",
  "setuptools.msvc.winreg",
  "setuptools.namespaces",
  "setuptools.namespaces.DevelopInstaller",
  "setuptools.namespaces.Installer",
  "setuptools.namespaces.flatten",
  "setuptools.namespaces.map",
  "setuptools.package_index",
  "setuptools.package_index.ContentChecker",
  "setuptools.package_index.Credential",
  "setuptools.package_index.EGG_FRAGMENT",
  "setuptools.package_index.EXTENSIONS",
  "setuptools.package_index.HREF",
  "setuptools.package_index.HashChecker",
  "setuptools.package_index.PYPI_MD5",
  "setuptools.package_index.PackageIndex",
  "setuptools.package_index.PyPIConfig",
  "setuptools.package_index.REL",
  "setuptools.package_index.URL_SCHEME",
  "setuptools.package_index._SOCKET_TIMEOUT",
  "setuptools.package_index.__all__",
  "setuptools.package_index._encode_auth",
  "setuptools.package_index._tmpl",
  "setuptools.package_index.configparser",
  "setuptools.package_index.decode_entity",
  "setuptools.package_index.distros_for_filename",
  "setuptools.package_index.distros_for_location",
  "setuptools.package_index.distros_for_url",
  "setuptools.package_index.egg_info_for_url",
  "setuptools.package_index.entity_sub",
  "setuptools.package_index.find_external_links",
  "setuptools.package_index.fix_sf_url",
  "setuptools.package_index.get_all_headers",
  "setuptools.package_index.htmldecode",
  "setuptools.package_index.http_client",
  "setuptools.package_index.interpret_distro_name",
  "setuptools.package_index.local_open",
  "setuptools.package_index.map",
  "setuptools.package_index.open_with_auth",
  "setuptools.package_index.parse_bdist_wininst",
  "setuptools.package_index.parse_requirement_arg",
  "setuptools.package_index.six",
  "setuptools.package_index.socket_timeout",
  "setuptools.package_index.ssl_support",
  "setuptools.package_index.strip_fragment",
  "setuptools.package_index.uchr",
  "setuptools.package_index.unique_everseen",
  "setuptools.package_index.unique_values",
  "setuptools.package_index.urllib",
  "setuptools.package_index.user_agent",
  "setuptools.py26compat",
  "setuptools.py26compat.import_module",
  "setuptools.py26compat.strip_fragment",
  "setuptools.py27compat",
  "setuptools.py27compat.get_all_headers",
  "setuptools.py27compat.linux_py2_ascii",
  "setuptools.py27compat.rmtree_safe",
  "setuptools.py27compat.six",
  "setuptools.py31compat",
  "setuptools.py31compat.TemporaryDirectory",
  "setuptools.py31compat._PY31",
  "setuptools.py31compat.__all__",
  "setuptools.py31compat.get_path",
  "setuptools.py31compat.unittest_main",
  "setuptools.py33compat",
  "setuptools.py33compat.Bytecode",
  "setuptools.py33compat.Bytecode_compat",
  "setuptools.py33compat.OpArg",
  "setuptools.py33compat.six",
  "setuptools.py36compat",
  "setuptools.py36compat.Distribution_parse_config_files",
  "setuptools.run_2to3_on_doctests",
  "setuptools.sandbox",
  "setuptools.sandbox.AbstractSandbox",
  "setuptools.sandbox.DirectorySandbox",
  "setuptools.sandbox.ExceptionSaver",
  "setuptools.sandbox.SandboxViolation",
  "setuptools.sandbox.UnpickleableException",
  "setuptools.sandbox.WRITE_FLAGS",
  "setuptools.sandbox._EXCEPTIONS",
  "setuptools.sandbox.__all__",
  "setuptools.sandbox._clear_modules",
  "setuptools.sandbox._execfile",
  "setuptools.sandbox._file",
  "setuptools.sandbox._needs_hiding",
  "setuptools.sandbox._open",
  "setuptools.sandbox._os",
  "setuptools.sandbox.builtins",
  "setuptools.sandbox.hide_setuptools",
  "setuptools.sandbox.map",
  "setuptools.sandbox.override_temp",
  "setuptools.sandbox.pushd",
  "setuptools.sandbox.run_setup",
  "setuptools.sandbox.save_argv",
  "setuptools.sandbox.save_modules",
  "setuptools.sandbox.save_path",
  "setuptools.sandbox.save_pkg_resources_state",
  "setuptools.sandbox.setup_context",
  "setuptools.sandbox.six",
  "setuptools.setup",
  "setuptools.site-patch",
  "setuptools.ssl_support",
  "setuptools.ssl_support.CertificateError",
  "setuptools.ssl_support.HTTPSConnection",
  "setuptools.ssl_support.HTTPSHandler",
  "setuptools.ssl_support.VerifyingHTTPSConn",
  "setuptools.ssl_support.VerifyingHTTPSHandler",
  "setuptools.ssl_support.__all__",
  "setuptools.ssl_support._certifi_where",
  "setuptools.ssl_support._dnsname_match",
  "setuptools.ssl_support.cert_paths",
  "setuptools.ssl_support.filter",
  "setuptools.ssl_support.find_ca_bundle",
  "setuptools.ssl_support.get_win_certfile",
  "setuptools.ssl_support.http_client",
  "setuptools.ssl_support.is_available",
  "setuptools.ssl_support.map",
  "setuptools.ssl_support.match_hostname",
  "setuptools.ssl_support.once",
  "setuptools.ssl_support.opener_for",
  "setuptools.ssl_support.ssl",
  "setuptools.ssl_support.urllib",
  "setuptools.unicode_utils",
  "setuptools.unicode_utils.decompose",
  "setuptools.unicode_utils.filesys_decode",
  "setuptools.unicode_utils.six",
  "setuptools.unicode_utils.try_encode",
  "setuptools.version",
  "setuptools.version.__version__",
  "setuptools.windows_support",
  "setuptools.windows_support.hide_file",
  "setuptools.windows_support.windows_only"
 ]
}